### DISCUSSION TITLE
Maximum Join Request

### DISCUSSION THREAD

**Discussion by JamshedAlamQaderi** [shepeliev/webrtc-kmp#95] [General]
Hi,
I'm trying to implement this library and I want know, how many users can connect in a single room?

**Comment by shepeliev** [comment]
Hi.

If you mean the sample app it's just a basic implementation and doesn't support multiuser connections.

**Reply by JamshedAlamQaderi** [reply]
Is there any plan to make this library more stable?

**Reply by shepeliev** [reply]
Sure. I'm working on it as much as I can dedicate time. Also, anybody can contribute too. If you know what could be improved PRs are wellcome :)
